### DISCUSSION TITLE
【0x06】SVG NFT 全面实践 | Web3.0 dApp 开发（六）

### DISCUSSION THREAD

**Discussion by leeduckgo** [NonceGeek/Move-Camp#38] [Projects]
> https://mp.weixin.qq.com/s/WvTFk3E6FjfHtXfp2uHkrw

Video:

> https://ke.qq.com/course/3616174?tuin=d34bd514&taid=12770514027621806

**Comment by gofenix** [comment]
wow!!!

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/6822558/158930150-833eb1fa-0b14-4880-a8e9-665105c0652a.png">
